### ISSUE TITLE
using Counters.Counter

### ISSUE THREAD

**Issue by efecarranza** [ConsenSys/scribble#174]
Hi,

When using OpenZeppelin's counters, scribble is unable to understand that for example, `current()` is a function and says Counters.Counter doesn't have property `current`.

```
using Counters for Counters.Counter;
Counters.Counter private _tokenIdCounter;
```

For example:

`///#if_succeeds old(tokenCounter.current()) + 1 == tokenCounter.current();`

Can we look into this?

Thank you

**Comment by blitz-1306** [comment]
Hello there. First of all, I guess `tokenCounter` should be `_tokenIdCounter` in the annotation.

I managed to reproduced the issue anyway with the following complete sample:
```solidity
library Counters {
    struct Counter {
        uint256 _value;
    }

    function current(Counter storage counter) internal view returns (uint256) {
        return counter._value;
    }

    function increment(Counter storage counter) internal {
        unchecked {
            counter._value += 1;
        }
    }
}

contract Test {
    using Counters for Counters.Counter;

    Counters.Counter private _tokenIdCounter;

    ///#if_succeeds old(_tokenIdCounter.current()) + 1 == _tokenIdCounter.current();
    function doSomething() public {}
}
```
Error message is following:
```
sample.sol:22:24 TypeError: Struct Counters.Counter doesn't have a field current
sample.sol:22:24:
    ///#if_succeeds old(_tokenIdCounter.current()) + 1 == _tokenIdCounter.current();
                        ^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by blitz-1306** [comment]
I've created PR to fix the issue: #177

**Comment by blitz-1306** [comment]
@efecarranza Please, check new version of Scribble: https://github.com/ConsenSys/scribble/releases/tag/v0.6.10. The bug should be fixed. Thanks for the collaboration.
